exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillConnectionImpl.java

```diff
@@ -231,13 +217,7 @@ public DrillClient getClient() {
 
   @Override
   public void setAutoCommit(boolean autoCommit) throws SQLException {
```

Comment:
Currently, `setAutoCommit()` override super class method but didn't do anything, is right?

Reply:
Yes, it is correct. Drill doesn't support transactions, so we shouldn't apply the logic from the parent method, but we also shouldn't throw an error as we did before to be compatible with [Python Database API Specification v2.0](https://www.python.org/dev/peps/pep-0249/#commit).